### ISSUE TITLE
Page showing remaining registrations

### ISSUE THREAD

**Issue by dorodere** [dorodere/komplett-normale-busliste#8]
Depends on #5 and partly #6.

After #5 has been implemented, it'd be nice to have a web page thought for a screen in public, which just shows a QR-code to the instance, the remaining count of possible registrations for the next two drives containing also the weekday names, and finally, also until when the user still has time to register (hence the #6 dep).

Unfortunately I believe this will require some kind of JavaScript/https://yew.rs to regularly fetch the data on available registrations, which will also require a new API route. Open to debate about whether that one should be protected through an API-ish key or not.

**Comment by Nitro-paul** [comment]
Fetching data will not require a lot of javaScript or a new api route. Using a little bit of Js to reload the page would do the trick.
